controllers/add_pods.go

```diff
@@ -38,13 +38,14 @@ type AddPods struct{}
 // Reconcile runs the reconciler's work.
 func (a AddPods) Reconcile(r *FoundationDBClusterReconciler, context ctx.Context, cluster *fdbtypes.FoundationDBCluster) *Requeue {
 	configMap, err := internal.GetConfigMap(cluster)
+	logger := log.WithValues("cluster", cluster.Name, "namespace", cluster.Namespace, "reconciler", "AddPods")
```

Comment:
```suggestion
	logger := log.WithValues("namespace", cluster.Namespace, "cluster", cluster.Name, "reconciler", "AddPods")
```
In most places we have namespace as the first pair (it shouldn't matter in the end).